src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -456,4 +460,4 @@ public net.minecraft.world.level.levelgen.feature.trunkplacers.TrunkPlacerType <
 protected net.minecraft.world.level.portal.PortalForcer f_77648_ # level
 public net.minecraft.world.level.storage.LevelResource <init>(Ljava/lang/String;)V # constructor
 private-f net.minecraft.world.level.storage.loot.LootPool f_79028_ # rolls
-private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
\ No newline at end of file
+private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
```

Comment:
Remove unnecessary change

Reply:
My bad I had a setting enabled that told idea to ensure every file ended with a new line, disabled that, and reverted the change (removed the blank new line).